SYCL/Assert/assert_in_simultaneous_kernels.cpp

```diff
@@ -10,7 +10,9 @@
 // too early. This is necessary because currently SYCL RT sets indirect access
 // flag for all kernels and the Level Zero runtime doesn't support deferred
 // release yet.
-// RUN: env SYCL_PI_LEVEL_ZERO_TRACK_INDIRECT_ACCESS_MEMORY=1 %GPU_RUN_PLACEHOLDER %t.out &> %t.txt || true
+// Suppress runtime from printing out error messages, so that the test can
+// match on assert message generated by the toolchains.
+// RUN: env SYCL_PI_LEVEL_ZERO_TRACK_INDIRECT_ACCESS_MEMORY=1 env SYCL_PI_SUPPRESS_ERROR_MESSAGE=1 %GPU_RUN_PLACEHOLDER %t.out &> %t.txt || true
```

Comment:
minor comment: I believe 'env' one time is enough. I.e. "env A=aa B=bb" instead of "env A=aa env B=bb"

Reply:
OK, done.